base/uk.ac.stfc.isis.ibex.ui.scriptgenerator/src/uk/ac/stfc/isis/ibex/ui/scriptgenerator/views/ScriptGeneratorView.java

```diff
@@ -213,19 +213,16 @@ private void doGitActions() {
 						"Could not update script definitions, because the remote git repository could not be reached. "
 						+ "You can still continue to use the existing script definitions, but they may be out of date.");
 			}
+			String message = "";
 			if (scriptGeneratorViewModel.updatesAvailable()) {
 				// Display prompt if new commits are available
-				int performMerge = MessageDialog.open(MessageDialog.CONFIRM,
-						DISPLAY.getActiveShell(),
-						"Error pulling repository",
-						scriptGeneratorViewModel.getPromptMessage(),
-						0,
-						"Keep local changes",
-						"Discard local changes and update");	
-				
-				if (performMerge == 1) {
-					scriptGeneratorViewModel.mergeOrigin();
```

Comment:
i think we can bin off all the actual git logic in the view model too

Reply:
I spent quite some time trying to actually find where the git logic code is, and couldn't seem to. Do you know where this git logic stuff is located? Good idea to remove it I think.

Reply:
Found the git stuff with the help of Thomas. Have removed it from python interface's "git_utils" as well.